modules/admin_manual/pages/release_notes.adoc

```diff
@@ -30,6 +31,130 @@
+`occ config:app:set core lock_timeout_default --value 1800`
+
+- _Maximum timeout for the locks (in seconds)_: Maximum lifetime of locks which is allowed to be set by calling the WebDAV Locks API
+`occ config:app:set core lock_timeout_max --value 86400`
+
+By default locks set in the web interface will expire after 30 minutes. The maximum lock time by default is one day.
+
+TIP: Please change the lock expiration settings according to your needs. Usually you will only need to change the default timeout as that applies to locks set in the web interface.
+
+TIP: The user-facing components in the web interface are disabled by default. Administrators can enable the feature by executing the following _occ_ command: `occ config:app:set files enable_lock_file_action --value yes`
+
+=== Changes to the ownCloud Marketplace
+
+The ownCloud Marketplace is opening up. With the release of Server 10.5, all apps available on the Marketplace (including ownCloud Enterprise apps) are available for download and installation via the Market app, respectively. This change facilitates the process of getting started with ownCloud Server and of evaluating Enterprise functionality. Additionally, it allows updates for all apps to be obtained from the ownCloud Marketplace and ensures running up-to-date versions of the extensions.
+
+=== Changes in Delivery
+
+In line with the changes to the Marketplace the deliverables for ownCloud Server have been unified. Previously there were different Tarball bundles, Docker images and Linux packages for the Community (bare minimum) and Enterprise (all supported apps) Editions.
+Starting with Server 10.5 there are the following bundles which are shipped via Tarball, Docker images and Linux packages:
+- minimal bundle for the Server and required components, semantically versioned (`ownCloud-10.5.0`)
+- complete bundle for the Server and all supported apps, including the Enterprise features, not semantically versioned as it always contains the latest versions of all supported apps (`ownCloud-complete-<date>`)
+
+The availability of Enterprise features now only depends on the license key.
+As part of this process, the former `enterprise_key` app has been deprecated and is not shipped anymore. New versions for all Enterprise apps have been released and included in the Server 10.5 complete bundle. These are needed to work with PHP 7.4 and the new licensing mechanism (see below).
+
+TIP: **When upgrading to Server 10.5 it is mandatory to upgrade all apps to the latest versions as well to ensure compatibility with the supported PHP versions and the new licensing mechanism. Using the Server 10.5 complete bundle for upgrading you will get the latest versions of all apps.**
+
+TIP: It is mandatory to disable the deprecated `enterprise_key` app when upgrading. A migration step will do this automatically during the upgrade procedure.
```

Comment:
The enterprise app was not disabled when I tested. ( I added because the app was not in the bundle)

Probably we have to say: 
You don't have to add the enterprise key app in the bundle.
In case you added there (in addition to community apps) then it won't be disabled automatically, please make it manually and retry the upgrade.
After the upgrade, the Enterprise key app is not necessary anymore it can be removed safely from the apps folder. 
(Well you can rewrite my comment better ;) )

Reply:
Mitigated by the dummy app